packages/databyss-ui/modules/Sidebar/Sidebar.js

```diff
@@ -133,6 +72,9 @@ const Sidebar = () => {
         </PagesLoader>
       )
     }
+    if (menuItem === 'search') {
```

Comment:
Let's move `menuItem` out of local state to `NavigationProvider`, and let's use the same route convention as the content area. So instead of `setMenuItem('search')` would do something like `NavigationProvider.navigateSidebar('/search')`. Then we won't have to pass 'setMenuItem` and `onClick` down through props, but instead can just consume `NavigationProvider` in child components where we need them.